.github/workflows/queue.yaml

```
@@ -0,0 +1,73 @@
+          load_submodules: "true"
+          with_nix: "true"
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Authenticate to Google Cloud
+        uses: google-github-actions/auth@v2
+        with:
+          credentials_json: ${{ secrets.GCP_SA_KEY }}
+      - name: Set up Cloud SDK # needed for gcloud to upload runners to GCS for caching
+        uses: google-github-actions/setup-gcloud@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      - run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:20: Double quote to prevent globbing and word splitting

(shellcheck)